apps/govern/components/VeOlas/index.tsx

```diff
@@ -23,53 +25,64 @@ const Title = styled.h1`
 `;
 
 export const VeOlasPage = () => {
-  const { isLoading, canWithdrawVeolas, canIncreaseAmountOrUnlock } = useFetchBalances();
+  const { isLoading, canWithdrawVeolas, canIncreaseAmountOrUnlock } =
+    useFetchBalances();
 
-  const [isCreateLockModalVisible, setIsCreateLockModalVisible] = useState(false);
+  const [isCreateLockModalVisible, setIsCreateLockModalVisible] =
+    useState(false);
   const [isIncreaseModalVisible, setIsIncreaseModalVisible] = useState(false);
 
   return (
-    <StyledMain>
-      <Card>
-        <Title>veOLAS</Title>
-        <Paragraph type="secondary" className="mt-8 mb-24">
-          veOLAS gives you voting power in Olas governance. Lock OLAS for longer periods to get more
-          veOLAS.
-        </Paragraph>
-        <Space size="middle" className="mb-16">
-          <Button
-            type="primary"
-            size="large"
-            disabled={isLoading || !!canWithdrawVeolas}
-            onClick={() => {
-              // if the user has veolas, then show the modal to increase the amount
-              // else show the modal to create a lock
-              if (canIncreaseAmountOrUnlock) {
-                setIsIncreaseModalVisible(true);
-              } else {
-                setIsCreateLockModalVisible(true);
-              }
-            }}
-          >
-            Lock OLAS for veOLAS
-          </Button>
+    <>
+      <Head>
```

Comment:
could you please add the whole meta block to govern app similar to one that we have on olas website or contribute or in bond app here apps/bond/pages/_app.jsx? currently govern app is not searchable in google :( I probably forgot to add that when created the app